packages/ckeditor5-ckbox/src/ckboxcommand.ts

```diff
@@ -299,6 +303,25 @@ export default class CKBoxCommand extends Command {
 		if ( selection.isCollapsed ) {
 			const selectionAttributes = toMap( selection.getAttributes() );
 			const textNode = writer.createText( linkName, selectionAttributes );
+
+			if ( writer && assetsCount > 1 ) {
```

Comment:
I don't think, `_insertLink` is the proper place to check `assetsCount > 1`.
For instance, `_insertAssets` takes the parameter `isLastAsset` - it does not check itself if it is the last one. It could take `isSingleAsset` as well and pass it here.